File: Code/App/v2/Laboratory/Scripts/App/Models/LabExaminationViewModel.js
function LabExaminationViewModel(data) {
    var self = this;

    this.PatientDieseaseId = ko.observable(data.PatientDieseaseId);

    this.PatientInfo = new PatientsModel(data.PatientInfo);
    this.LabComment = new
        LabExaminationCommentViewModel(data.LabComment);
    this.examinationDescriptionLab= ko.observable();

};

File: Code/App/separateDB/Laboratory/ViewModels/LabExaminationViewModel.cs
using Laboratory.ViewModels;
using RepositoryClasses.Models;

namespace Laboratory.ViewModels
{
    public class LabExaminationViewModel
    {
        public PatientsModel PatientInfo { get; set; }
        public LabExaminationCommentViewModel LabComment { get; set; }
        public int PatientDieseaseId { get; set; }
    }
}